CalibTracker/SiStripCommon/plugins/ShallowGainCalibration.cc

```diff
@@ -36,17 +34,17 @@ void ShallowGainCalibration::
 produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
   std::auto_ptr<std::vector<int> >            trackindex    ( new std::vector<int>            );
   std::auto_ptr<std::vector<unsigned int> >   rawid         ( new std::vector<unsigned int>   );
-  std::auto_ptr<std::vector<float>  >         localdirx     ( new std::vector<float>          );
-  std::auto_ptr<std::vector<float>  >         localdiry     ( new std::vector<float>          );
-  std::auto_ptr<std::vector<float>  >         localdirz     ( new std::vector<float>          );
+  std::auto_ptr<std::vector<double> >         localdirx     ( new std::vector<double>         );
+  std::auto_ptr<std::vector<double> >         localdiry     ( new std::vector<double>         );
+  std::auto_ptr<std::vector<double> >         localdirz     ( new std::vector<double>         );
```

Comment:
out of curiosity was there some study done to say that floats were not sufficient?